packages/react/src/FilteredActionList/FilteredActionList.tsx

```diff
@@ -109,6 +111,61 @@ export function FilteredActionList({
 
   useScrollFlash(scrollContainerRef)
 
+  function MappedActionList(item: ItemInput) {
+    const {
+      description,
+      descriptionVariant,
+      sx,
+      text,
+      variant,
+      disabled,
+      trailingVisual: TrailingVisual,
```

Comment:
👋 

the TitleCase is so that we can render them as a component in JSX

prop accepted as `leadingVisual`, rendered in the body as `<LeadingVisual/>`, it would have been nicer if the prop was already TitleCased to signal that it accepts a component (and not a string, for example), but probably not the correct time to change that here